poc-07.py

```diff
@@ -14,7 +14,7 @@ class Message(object):
     def __init__(self, uid=None, body=None, flags=None):
         self.uid = uid
         self.body = body
-        self.flags = {'read': False, 'important': False}
+        self.flags = {'read': None, 'important': None, 'deleted': None}
```

Comment:
Actually, no. That's wrong.
Marking a message as READ (True, False) is wanted.
We want additional methods for **changes**.